packages/common/src/components/PostCard/PostCard.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled'
+import { Story, Meta } from '@storybook/react'
+import { PostCard, PostCardProps } from './PostCard'
+
+export default {
+  title: 'PostCard',
+  component: PostCard,
+  argTypes: {
+    categories: {
+      control: false,
+    },
+  },
+  decorators: [
+    Story => (
+      <StyledContainer>
+        <Story />
+      </StyledContainer>
+    ),
+  ],
+} as Meta
+
+const StyledContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  position: absolute;
+  top: 60px;
+  left: 0;
+  right: 0;
+  margin-left: auto;
+  margin-right: auto;
+`
+
+const Template: Story<PostCardProps> = args => <PostCard {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  categories: ['Travel'],
```

Comment:
categories are going to be capitalized? I think they were in lowercase elsewhere.